ghost/core/test/e2e-frontend/helpers/get.test.js

```diff
@@ -98,6 +99,190 @@ describe('e2e {{#get}} helper', function () {
         locals = {root: {_locals: {}}};
     });
 
+    describe('Filter optimisation', function () {
```

Comment:
Would it be worth adding a test for multiple id negations? ie `id:-abcdef+id:-ghijkl`?

Reply:
IDK if this is possible but still